docs/lib/datastore/fragments/native_common/setup-auth-rules.md

```diff
@@ -0,0 +1,123 @@
+
+* Create/update/delete mutations are private to the owner.
+```graphql
+type YourModel @model @auth(rules: [{ allow: owner }]) {
+  ...
+}
+```
+
+* Allow others to update and read, but not delete.
+```graphql
+type YourModel @model @auth(rules: [{ allow: owner,
+                                   operations: [create, delete]}]) {
+  ...
+}
+```
+
+### Static Group Authorization
+The following are commonly used patterns for static group authorization.  For more information on how to tune these examples, please see the [CLI documentation on static group authorization](~/cli/graphql-transformer/auth.md#static-group-authorization).
+
+* Users belonging to the "Admin" group can CRUD (create, read, update, and delete), others can not access anything.
+```graphql
+type YourModel @model @auth(rules: [{ allow: groups,
+                                      groups: ["Admin"] }]) {
+  ...
+}
+```
+
+* Users belonging to the "Admin" group can CRUD, others query and update.
+```graphql
+type YourModel @model @auth(rules: [{ allow: groups,
+                                       groups: ["Admin"],
+                                   operations: [create, delete] }]) {
+  ...
+}
+```
+
+### Owner and Static Group Combined
+The following are commonly used patterns for combining owner and static group authorization.  For more information on how to tune these examples, please see the [CLI documentation on static group authorization](~/cli/graphql-transformer/auth.md#static-group-authorization).
+
+* Users have their own data, but users who belong to the `Admin` group have access to their data and anyone else in that group.  Users in the `Admin` group have the ability to make mutation on behalf of users not in the `Admin` group
+```graphql
+type YourModel @model @auth(rules: [{ allow: owner },
+                                      { allow: groups, groups: ["Admin"]}]) {
+  ...
+}
+```
+
+### Public Authorization
+The following are commonly used patterns for public CRUD authorization.  For more information on how to tune these examples, please see the [CLI documentation on static group authorization](~/cli/graphql-transformer/auth.md#static-group-authorization#public-authorization).
+
+* Auth provider is API Key, and all data is public
+```graphql
+type YourModel @model @auth(rules: [{ allow: public }]) {
+  ...
+}
+```
+
+* Auth provider is IAM, and all data is public CRUD
+```graphql
+type YourModel @model @auth(rules: [{ allow: public, provider: iam }]) {
+  ...
+}
+```
+
+### Private Authorization
+The following are commonly used patterns for private authorization.  For more information on how to tune these examples, please see the [CLI documentation on static group authorization](~/cli/graphql-transformer/auth.md#static-group-authorization#private-authorization).
+
+* Cognito user pool authenticated users can CRUD all posts, regardless of who created it.  Guest users do not have access.
```

Comment:
You use the term "Guest user" but don't use it in the "public" section. Choose either "Guest" or "Public" and use throughout the doc consistently.

Reply:
Sorry, can you be more specific, I'm having trouble understanding what you're suggesting here.